### PR TITLE
Add some improvements

### PR DESCRIPTION
**Pull request by pepness**
-Fix wrong paths
-Add module author
-Change display name
-Add short description

### CODE REVIEW

**Comment by juneau001** [comment]
Thanks @pepness I appreciate it.  I apologize for your extra work...I had pushed more changes on Saturday after I had already opened this PR to repair the incorrect paths, but I had not added module author, changed display name, etc.  Thanks for making those adjustments!  Your time and assistance is certainly appreciated.

**Review by juneau001 — APPROVED** [review]
Thanks for making these changes.  All of your modifications look good and I appreciate it.

File: nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps
Comment (juneau001) [inline nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps:1]: Are these changes required for the PR?  It seems like a lot of changes to the file that are not related to any work done within this PR.  However, it could be due to my working on the master branch as opposed to delivery.

Comment (pepness) [reply nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps:1]: Only the last 3 commits are mine, the first 4 are from master. I created a new branch from my master and then rebase your PR into my newly created branch. The problem is that when I created the new branch it was already a few commits ahead of your PR. Please just use the last 3 commits and discard the first 4 commits. I will be more careful next time.

Comment (juneau001) [reply nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps:1]: No problem at all, I appreciate your assistance.  I'll get your enhancements worked into the PR.  Thanks

**Comment by pepness** [comment]
Hi @juneau001 , I add some improvements, this time there are no extra commits. 

Please tell me how else can I help you get this PR into shape.

**Comment by juneau001** [comment]
> Hi @juneau001 , I add some improvements, this time there are no extra commits.
> 
> Please tell me how else can I help you get this PR into shape.

Thanks @pepness I appreciate it.  I will look at the improvements and apply them.  I have been working with @matthiasblaesing on getting the PR into shape also, and it was decided that this will wait until NB 19 since it is too large of a change to make at this stage.   Matthias sent me a patch to repair a failing commit test and I will apply that also so that this can be ready to go for NB 19.  I also plan to blog about it so that others can test from my repository before it goes to NB19.  Thanks again I appreciate your time and assistance!

**Review by juneau001 — APPROVED** [review]
These changes look great @pepness thanks.  I will add them to this branch to be included in the NETBEANS-4035 PR.
